airflow/www/gunicorn_config.py

```diff
@@ -37,3 +42,18 @@ def on_starting(server):
 
     # Load providers before forking workers
     ProvidersManager().connection_form_widgets
+
```

Comment:
This one is a bit tricky - but I think there shoudl also be a way to pass the information from the `--open-telemetry` flag of the `webserver` herre. Likely it can be done by setting an environment variable in the "webserver" command.

Something like:

```
os.environ["AIRFLOW_ENABLE_GUNICORN_OPENTELEMETRY"] = "true"
```

Then you could check it in post_fork() method and run the code if the variable is set.

Reply:
![webserver_command](https://user-images.githubusercontent.com/78446940/151338821-493bc187-25d3-4c6e-956f-d3ec900f5d23.PNG)
![guinicorn](https://user-images.githubusercontent.com/78446940/151338839-9303ded1-1533-4286-a052-04821335a40b.PNG)

Hi @potiuk, is this what you mean?

Reply:
Yes. Something like that.should work I think. Needs testing though - for example you could `raise Exception()` in the "if" part and then, if the variable is properly set and passed to the gunicorn, all gunicorns would fail.

Reply:
ok I tested it by raising Exception in the if clause, and all gunicorns failed, so I think its ok